app/api/event_copyright.py

```diff
@@ -71,20 +73,20 @@ def before_create_object(self, data, view_kwargs):
 
 class EventCopyrightDetail(ResourceDetail):
 
-    def before_patch(self, args, kwargs):
+    def before_patch(self, args, kwargs, data):
```

Comment:
Might be. Because even the function definition for before_patch makes sense to define data as a parameter. Check [this](https://github.com/miLibris/flask-rest-jsonapi/blob/master/flask_rest_jsonapi/resource.py#L502).